Matched "crisp-edges" and "pixelated" image-rendering to W3C spec (#24736)
mdn/content: files/en-us/web/css/image-rendering/index.md

- `high-quality` {{Experimental_Inline}}
  - : Identical to `smooth`, but with a preference for higher-quality scaling. If system resources are constrained, images with `high-quality` should be prioritized over those with any other value, when considering which images to degrade the quality of and to what degree.
- `crisp-edges`
  - : The image must be scaled with an algorithm that preserves contrast and edges in the image, and which does not smooth colors or introduce blur to the image in the process. This is intended for images such as pixel art or line drawings.
- `pixelated`

>   - : The image must be scaled with the "nearest neighbor" or similar algorithm, to preserve a "pixelated" look as the image changes in size.

Reviewer: ```suggestion
  - : The image is scaled with the "nearest neighbor" or similar algorithm, preserving a "pixelated" look as the image changes in size.
```
